### DISCUSSION TITLE
Display the date last updated and what changed

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#256] [General]
On https://zosopentools.github.io/meta/#/Latest would it be possible to provide a bit more info such as the date last changed and a summary of what changed?

**Comment by MikeFultonDev** [comment]
I think we are still coming up short on this one... Theoretically we could pull the PR title but there are a lot of poor PR titles - maybe we need to get a little more rigorous about that @IgorTodorovskiIBM ? Or have an easy way to say 'ignore this PR - it's minor' ?
